controllers/front/ProductController.php

```diff
@@ -514,7 +514,7 @@ protected function assignPriceAndTax()
             }
         }
 
-        $product_price = $this->product->getPrice(Product::$_taxCalculationMethod == PS_TAX_INC, $id_product_attribute);
+        $product_price = $this->product->getPrice(Product::$_taxCalculationMethod == PS_TAX_INC, $id_product_attribute, 6, null, false, false);
```

Comment:
Why 6 ? There is no constant for this ?

Reply:
No, there isn't.
The original method contain this value without constant, as you can see at the picture.
![getPrice](https://user-images.githubusercontent.com/1076017/74722921-c5559800-524a-11ea-86dd-a7e95b9ee9a0.jpg)

Reply:
OK. Thanks